From 99a637fb3f98e9008f2ca676e85f4dcd6e211667 Mon Sep 17 00:00:00 2001
From: "lukeroweprivate@gmail.com" <lukeroweprivate@gmail.com>
Date: Tue, 19 Jan 2016 20:14:32 -0800
Subject: [PATCH] Did a fix for Eclipse

---
 .project | 12 ++++++++++++
 1 file changed, 12 insertions(+)
 create mode 100644 .project

diff --git a/.project b/.project
new file mode 100644
index 0000000..7675629
--- /dev/null
+++ b/.project
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>RemoteSystemsTempFiles</name>
+	<comment></comment>
+	<projects>
+	</projects>
+	<buildSpec>
+	</buildSpec>
+	<natures>
+		<nature>org.eclipse.rse.ui.remoteSystemsTempNature</nature>
+	</natures>
+</projectDescription>
-- 
2.7.0.windows.1

